src/Publisher.php

```diff
@@ -92,6 +99,10 @@ public function generatePageResponse($url)
             );
             $app = $this->getHTTPApplication();
             $response = $app->handle($request);
+
+            if ($this->config()->get('add_timestamp') === true) {
+                $response->setBody(str_replace('</html>', "<!-- " . DBDateTime::now() . " -->\n</html>", $response->getBody()));
```

Comment:
`DBDateTime::now()` needs to be formatted appropriately eg: `DBDateTime::now()->Full()` or maybe `DBDateTime::now()->Rfc2822()`

Reply:
You also need to bring this line below 120 chars long